02/Task02.js

```diff
@@ -1,15 +1,33 @@
-import React from 'react';
-import Welcome from './../src/components/Welcome';
-import withPopup from './../src/hoc/withPopup';
+import React, { useState } from "react";
+import Welcome from "./../src/components/Welcome";
+import withPopup from "./../src/hoc/withPopup";
+import Popup from "../src/components/Popup";
 
-const ComponentWithPopup = withPopup(Welcome);
+const ComponentWithPopup = withPopup(Welcome())(Popup);
 
-const Task02 = () => (
+const Task02 = () => {
+  const [openPopup, setOpenPopup] = useState(true);
+
+  return (
     <section>
-        <h2>Task 02</h2>
-        <ComponentWithPopup />
+      <h2>Task 02</h2>
+      <ComponentWithPopup
+        isOpen={openPopup}
+        message={Welcome()}
```

Comment:
Skoro już w linii 6 przekazujesz `Welcome` to czy tutaj też jest to potrzebne?
Powielamy ten sam element - powinniśmy jakoś inaczej do tego podejść, aby nie robić tego 2 razy ;)